docs/howto.md

```diff
@@ -198,6 +200,8 @@ int main() {
 
 In this example, you can change the order of the fields in either `my_row` or your database query freely, as the fields are identified based on their name, and not based on their index in the tuple.
 
+ > **Note:** you may use `BOOST_FUSION_ADAPT_STRUCT` and all of the `Boost.Fusion` adaptation mechanisms for structures, but it is recommended to use `Boost.Hana` so as a faster and modern approach.
```

Comment:
> but it is recommended to use `Boost.Hana` so as a faster and modern approach.

but it is recommended to use `Boost.Hana` for a faster and more modern approach.

Reply:
Thanks, Michael!